products/health-care/appointments/va-online-scheduling/express-care-research/express-care-conversation-guide.md

```diff
@@ -0,0 +1,116 @@
+# Express Care Conversation Guide
+
+## Moderator logistics
+
+Use [#feedback-backchannel](https://dsva.slack.com/messages/C40B45NJK/details/) in Slack for real-time feedback from observers.*
+
+*Before the session, send out the [observer instructions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/during-research/howto-observer-instructions.md) to your observers (Slackbot will do this for you if you type "observer instructions" into #feedback-backchannel).*
+
+*In the Attendees section, make sure everyone except the participant is on mute*
+
+*In the Audio section, click the drop-down arrow and make sure "Play Entry/Exit Chimes" is unchecked*
+
+*Check your [screensharing setup]*
+
+*When the participant is ready, begin the session with the following intro*
+
+## Intro - 5 minutes
+
+Thanks for joining us today! My name is Melissa and I also have some colleagues on the line observing and taking notes. Today we're going to talk about scheduling Telehealth, or virtual medical, same-day appointments with a VA provider. I am going to ask you a series of questions and we'll take a look at a prototype to help my team understand how make this process easier for Veterans.
```

Comment:
sentence edit - last sentence _"help my team understand how **to** make this process easier for Veterans."_